component/src/test/java/com/fioalpha/poc/component/ExampleUnitTest.kt

```diff
@@ -1,8 +1,7 @@
 package com.fioalpha.poc.component
 
-import org.junit.Test
-
 import org.junit.Assert.*
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/fioalpha/CaseStudy/blob/6ce9cd99c0c1ee323fc1eba2aae62b43356b0111/component/src/test/java/com/fioalpha/poc/component/ExampleUnitTest.kt#L3"></span>Wildcard import (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>